### DISCUSSION TITLE
Ability to restrict the region a codespace deploys to within an org

### DISCUSSION THREAD

**Discussion by chris-bateman** [community/community#43671] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

Add the ability to create a policy that restricts which region users can start a codespace in.
This ticks a few compliance rules companies may have.

**Comment by cwndrws** [comment, marked as answer]
Hi! Thanks for writing in! We don't currently support this, but it's a good idea and we'll consider adding support for this in the future.

**Comment by chris-bateman** [comment]
@cwndrws Any progress 1 year later?
